Communication/Packets/Incoming/Avatar/SaveWardrobeOutfitEvent.cs

```diff
@@ -25,22 +25,8 @@ public Task Parse(GameClient session, ClientPacket packet)
         var gender = packet.PopString();
         look = _figureDataManager.ProcessFigure(look, gender, session.GetHabbo().GetClothing().GetClothingParts, true);
 
-        using (var connection = _database.Connection())
-        {
-            int rows = connection.Execute("SELECT null FROM `user_wardrobe` WHERE `user_id` = @id AND `slot_id` = @slot",
-                new { id = session.GetHabbo().Id, slot = slotId });
+        _userWardrobeManager.SaveOutfit(session.GetHabbo().Id, slotId, gender, look);

```

Comment:
Task needs to be awaited. Add await in front and make method async.

Reply:
```
public interface IPacketEvent
{
    Task Parse(GameClient session, ClientPacket packet);
}
```

That would require updating the underlying interface for packet events.   I made an alternative change, however that may do the trick.

```
        Task.Run(async () => await _userWardrobeManager.SaveOutfit(session.GetHabbo().Id, slotId, gender, look));

```

Reply:
Updated after a misunderstanding 👍